docs/keyConcepts.md

```diff
@@ -6,9 +6,9 @@ sort: 22
 
 ## Data model
 
-### What is a metric
+### What is a metric?
 
-Simply put, `metric` is a numeric measure or observation of something.
+Simply put, the `metric` is a numeric measure or observation of something.
```

Comment:
it is `a metric`, so far. Unless you can explain why `the`

Reply:
In both cases, if something was discussed previously (in 1-2 previous sentences it is usually referred to as "the" - though it is uncomfortable to read and might remove it completely.